README.md

```diff
@@ -1,86 +1,130 @@
+  - [SCSS](https://sass-lang.com) with [BEM](https://en.bem.info/methodology/css/) methodology
+  - Responsive with Desktop first priority
+  - Utilize media queries, SCSS mixing
+  - [Ant Design](https://ant.design) Library of pre-built React components
 
-#### Note
-The features of the Example Client are subject to change with future versions and as decisions are made on the DSNP SDK.
+* Testing:
+  - Tests written in [Jest](https://jestjs.io) and [Enzyme](https://enzymejs.github.io/enzyme/)
+  - CI/CD done with [Github Actions](https://github.com/features/actions)
 
-### Docker
-The Docker container is set up in such a way that the static-server will serve the example-client.
+*Note: Enzyme is [not ready for React 17
+yet](https://github.com/enzymejs/enzyme/issues/2429), so we're using
+an unofficial version and installing packages with `npm i
+--legacy-peer-deps` if using node v7.  We will change to an official
+version of Enzyme when a compatible released version is available.*
 
-To the example client and static server in docker run the following commands:
+## Deployment
 
-- `docker build --build-arg REACT_APP_UPLOAD_HOST="" --build-arg REACT_APP_CHAIN_ID={REACT_APP_CHAIN_ID_VALUE} --build-arg  REACT_APP_CHAIN_NAME={REACT_APP_CHAIN_NAME_VAUE}  --build-arg REACT_APP_CHAIN_HOST={REACT_APP_CHAIN_HOST} --build-arg REACT_APP_TORUS_BUILD_ENV={REACT_APP_TORUS_BUILD_ENV_VALUE} . -t example-client`
-- `docker run  --init --rm  -p 8080:8080 -v {name_for_volume}:/app/static-server/public example-client`
+The following steps should get you up and running quickly (assuming a
+working knowledge of NPM and Git):
 
-#### Docker Builds
-Docker image will be pushed from our [github repo](https://github.com/LibertyDSNP/example-client) to our [dockerhub](https://hub.docker.com/r/dsnp/example-client) when a new
-git tag is created.
+* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
 
-Tags will have the following format - `docker/*` i.e. `docker/v1.0.0`.
+* Install the correct npm and node version.  We recommend using asdf: `asdf install`
 
-***
-## Project Setup
-Follow this quick start guide to get up and running. This guide assumes a working knowledge of npm and git. Make sure to check out [CONTRIBUTING.md](CONTRIBUTING.md) as well.
-* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
-* Install the correct npm and node version. Recommend using asdf: `asdf install`
-* Install modules: `npm install`
+* Install the modules: `npm install`
   * Install Static Server modules `cd ./static-server && npm install`
+
 * Copy `.env.example` to `.env` and edit as needed.
-* Start the Static Server `cd ./static-server && npm run start`
+
+* Start the Static Server: `cd ./static-server && npm run start`
+
 * Start up site locally: `npm run start`
-* Visit http://localhost:3000 (or you configured port)
+
+* Visit http://localhost:3000 (or whatever your configured port is)
```

Comment:
Should we include that the user needs to install metamask and give instructions on the metmask setup?  If so, this seems like a good place to document that.

Reply:
Definitely include Metamask extension or Torus wallet as a dependency, but link to their websites rather than duplicating instructions that may go out of date.